libs/docs/src/docs/2.2.0/create-devfiles-with-templates.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Creating devfiles with templates
+description: Creating devfiles with templates
+---
+
+Most dev tools can utilize the devfile registry to 
+fetch the stacks needed to begin development on projects. 
+Sometimes, however, you might need to create a devfile 
+from scratch or to create your own devfile template(s). 
+This guide runs through the process, starting from a minimum devfile and 
+building sample templates for common use cases.
+
+## Prerequisites
+
+- [Versions](./versions)
+- [Metadata](./metadata)
```

Comment:
Will these links work? These topics are under the `/docs/no-version` folder not the current version specific location

Reply:
@kim-tsao Good point, though it looks to work as it seems no-version content get placed under all versions on build.